planning/robot_model_loader/src/robot_model_loader.cpp

```diff
@@ -51,11 +51,12 @@ robot_model_loader::RobotModelLoader::RobotModelLoader(const Options &opt)
   configure(opt);
 }
 
-robot_model_loader::RobotModelLoader::~RobotModelLoader()
-{
+robot_model_loader::RobotModelLoader::~RobotModelLoader() {
   model_.reset();
   rdf_loader_.reset();
   kinematics_loader_.reset();
+
+  ROS_DEBUG( "Robot model loader destructor" );
```

Comment:
I still think some user feedback and debug messages are useful, and they are easy to turn off using the rosconsole yaml configuration files. This is an underutilized feature of ROS, so I wrote up a blog post a while back about it: http://dav.ee/blog/notes/archives/898

However in this PR I think there are way too many. @v4hn is right that probably none of them are needed.